en-US/firefox-ios.xliff

```diff
@@ -1908,13 +1918,18 @@
         <note>Title for the wallpaper onboarding page in our Onboarding screens. This describes to the user that they can choose different wallpapers.</note>
       </trans-unit>
       <trans-unit id="Onboarding.Welcome.Description.v106" xml:space="preserve">
-        <source>Firefox puts people over profits and defends your privacy by default.</source>
-        <target>Firefox puts people over profits and defends your privacy by default.</target>
+        <source>Firefox puts people over profits and defends your privacy as you browse. </source>
+        <target>Firefox puts people over profits and defends your privacy as you browse. </target>
         <note>String used to describes the description of what Firefox is on the welcome onboarding page for 106 version in our Onboarding screens.</note>
       </trans-unit>
+      <trans-unit id="Onboarding.Welcome.Link.Action.v106" xml:space="preserve">
+        <source>Learn more in our privacy notice</source>
+        <target>Learn more in our privacy notice</target>
+        <note>String used to describe the title of link button is on the welcome onboarding page for 106 version in our Onboarding screens.</note>
+      </trans-unit>
       <trans-unit id="Onboarding.Welcome.Title.v106" xml:space="preserve">
```

Comment:
Also needs a new string ID since the string changes a lot (and so that localizers get notified of the change on Pontoon).
Also better to use a placeholder instead of the app name (and call that out in comments)